templates/client-config-configmap.yaml

```diff
@@ -1,6 +1,6 @@
+{{- if (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
```

Comment:
Is this change intentional?

Reply:
Yes, we need to move all comments inside the `if` statement. Otherwise the template gets rendered (with just the comments) and we can't do the same helm 3 error assertion. Since some templates have comments and some don't, it's better to standardize on all templates having comments inside the enclosing `if` statements so they always result in the same helm 3 error.

Reply:
Cool, makes sense. Thanks for clarifying!